charts/multitenancy/pols/add-network-policy.yaml

```diff
@@ -26,6 +25,7 @@ spec:
       kind: "NetworkPolicy"
       name: "default-deny"
       namespace: "{{request.object.metadata.name}}"
+      synchronize: true
```

Comment:
You need to update it here as well - https://github.com/nirmata/kyverno-policies/blob/main/multitenancy/add-network-policy.yaml